Tests/SwiftStdlibTests/DictionaryExtensionsTests.swift

```diff
@@ -163,4 +163,31 @@ final class DictionaryExtensionsTests: XCTestCase {
             [6: ["Bryant"], 5: ["James"], 4: ["Wade", "John"], 0: ["", ""]])
         XCTAssertEqual(Dictionary(grouping: array3, by: \String.count), [:])
     }
+
+    func testGetByKeys() {
+        let dict = ["James": 100,
+                    "Wade": 200,
+                    "Bryant": 500,
+                    "John": 600,
+                    "Jack": 1000]
+        let picked = dict.pick(keys: ["James", "Wade", "Jack"])
+        let empty1 = dict.pick(keys: ["Pippen", "Rodman"])
+        let empty2 = dict.pick(keys: ["Michael", "Joe", "Christian"])
```

Comment:
No need for `empty2`, you've already tested the scenario with `empty1`. What maybe you should have is a mixed one, where some kys are found and others are not.